core/src/main/java/org/opensearch/sql/storage/Table.java

```diff
@@ -16,6 +16,18 @@
  */
 public interface Table {
 
+  /**
+   * Check if current table exists.
+   * @return true if exists, otherwise false
+   */
+  boolean isExist();
```

Comment:
```suggestion
  boolean exists();
```
would be more conventional in Java.